docs/client.md

```diff
@@ -0,0 +1,53 @@
+# pb-cli specification
+
+## Overview
+
+`pb-cli` is a command line tool which communicate with `pb-go` backend. Although user can just upload data to pastebin directly with `curl`, we still provide a tools to help user easily use our advanced feature such as burn-after-reading and private share.
+
+We guarantee everything that worked with `pb-cli`, also worked with `curl`.
+
+## Configuration
+
+You can use `-c file` to provide a config file. If you didn't provide one, it will find config file called `.pb-cli.toml` from current directory then home directory.
```

Comment:
remove `.pb-cli.toml` to make it visible. We need to tell user not to forget it.

Reply:
try not to hide this file, use file name as `pb-cli.yaml` , like this.  Only search in current folder, don't search users folder since we need to consider Windows User.